compute/admin_guide/authentication/oidc.adoc

```diff
@@ -199,9 +206,9 @@ After you register Prisma Cloud as an Open ID Connect (OIDC) application in Azur
 
 .. *Client ID*: The Application ID for the client you have from Azure AD.
 .. *Client Secret*: The secret for the client that you created for the application and stored safely for later use.
-.. *Issuer URL*: The endpoint of the application registered in Azure AD, for example https://login.microsoftonline.com/`<TENANT_ID>`/v2.0
+.. *Issuer URL*: The endpoint of the application registered in Azure AD, for example *\https://login.microsoftonline.com/<TENANT_ID>/v2.0*
 .. *Group scope*: Leave this field blank.
-.. *Group claim*: The name of the group you created in Azure AD.
+.. *Group claim*: These are the groups explicitly assigned to the application that include the user as a member. The value is populated automatically.
```

Comment:
It's still not clear that we need the group claim key name - 'groups' here

Reply:
@rcaballeromx please fix this as Tomer explained. What the customer needs to set here is the key, which is just explicitly 'groups', without any specific group name, these are populated automatically.

Reply:
@grevach I rephrased to emphasize the value customers need to use.